pkg/segment/query/metadata/metareader_test.go

```diff
@@ -36,7 +36,7 @@ func Test_readWriteMicroIndices(t *testing.T) {
 	segKey := segDir + "test"
 	blockSummariesFile := structs.GetBsuFnameFromSegKey(segKey)
 	numBlocks := 10
-	_, blockSummaries, _, _, allBmh, allColsSizes := writer.WriteMockColSegFile(segKey, numBlocks, 30)
+	_, blockSummaries, _, _, allBmh, allColsSizes := writer.WriteMockColSegFile(segKey, segKey, numBlocks, 30)
```

Comment:
in line 34, rename `segDir` to be `dataDir`
and have a specific segBaseDir param, which is one level higher than segkey, otherwise someone else copies this code and may have wrong assumptions